spec/dummy/app/avo/actions/dummy_action.rb

```diff
@@ -1,7 +1,18 @@
 class DummyAction < Avo::BaseAction
   self.name = "Dummy action"
   self.standalone = true
-  self.visible = ->(resource:, view:) do
+  self.visible = -> do
```

Comment:
warning users about this change

Reply:
Could you also update the docs and the upgrade guide?

Reply:
sure